pkg/scalers/mysql_scaler.go

```diff
@@ -0,0 +1,200 @@
+package scalers
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/go-sql-driver/mysql"
+	"k8s.io/api/autoscaling/v2beta1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/metrics/pkg/apis/external_metrics"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+const (
+	mySQLMetricName      = "queryValue"
+	defaultMySQLPassword = ""
+)
+
+type mySQLScaler struct {
+	metadata   *mySQLMetadata
+	connection *sql.DB
+}
+
+type mySQLMetadata struct {
+	dsnConnStr string // Database connection string
```

Comment:
As mentioned in https://github.com/kedacore/keda-docs/pull/72, I'd presonally rename this to `connectionString`